server/service/src/sync/program_schemas/patient_ui_schema.json

```diff
@@ -150,10 +92,23 @@
                   "options": {
                     "conditionField": "birthPlace.region",
                     "conditionalValues": {
-                      "Western": ["Middle Fly", "North Fly District", "South Fly"],
+                      "Western": [
+                        "Middle Fly",
+                        "North Fly District",
+                        "South Fly"
+                      ],
                       "Gulf": ["Kerema District", "Kikori District"],
-                      "Central": ["Abau District", "Goilala District", "Kairuku-Hiri District", "Rigo District"],
-                      "National Capital District": ["Moresby North-East", "Moresby North_West", "Moresby South"],
+                      "Central": [
+                        "Abau District",
+                        "Goilala District",
+                        "Kairuku-Hiri District",
+                        "Rigo District"
+                      ],
+                      "National Capital District": [
+                        "Moresby North-East",
+                        "Moresby North_West",
+                        "Moresby South"
+                      ],
```

Comment:
Can we avoid these formatting changes if possible? (Mine reverts back to the original when I just save it)